packages/tinacms/src/auth/TinaCloudProvider.tsx

```diff
@@ -93,8 +92,14 @@ export const AuthWallInner = ({
             ...otherModalActions,
             {
               action: async () => {
-                setActiveModal(null)
-                setEdit(false)
+                // This does not work it looks like we have somehow getting two contexts
+                // console.log({ setEdit })
+                // setEdit(false)
+                // setActiveModal(null)
+
+                // This is a temp fix
+                setEditing(false) // set editing just sets the local storage
```

Comment:
Is there a separate bug card for tracking the actual issue?  Just want to make sure the "temp fix" doesn't become the "permanent fix" because we lost track of it.  Having two contexts should be a thing of the past...

Reply:
Good idea! https://github.com/tinacms/tinacms/issues/2081